Core/Core/Extensions/NSPersistentContainerExtensions.swift

```diff
@@ -31,15 +31,26 @@ extension NSPersistentContainer {
         }
 
         container.loadPersistentStores { _, error in
-            if let error = error {
-                fatalError(error.localizedDescription)
+            guard error == nil else {
+                container.destroy() // ignore migration conflicts
```

Comment:
Do you think it would be helpful to try to delete any existing`.sql` files as well?  Not sure if that would be helpful or not.  I read somewhere about errors in `container.loadPersistenstores` being because of a corrupt sql file.  I'll have to dig up the link.

Reply:
Yeah I think I know what you're talking about. Something to do with the journal files right? I think that would be worth looking into at some point.